flink-connectors/flink-connector-kinesis/src/main/resources/META-INF/NOTICE

```diff
@@ -17,6 +17,10 @@ This project bundles the following dependencies under the Apache Software Licens
 - com.amazonaws:aws-java-sdk-cloudwatch:1.12.276
 - com.amazonaws:dynamodb-streams-kinesis-adapter:1.5.3
 - com.amazonaws:jmespath-java:1.12.276
+- com.fasterxml.jackson.core:jackson-annotations:jar:2.13.2
```

Comment:
I believe the `:jar` bit at the end is redundant and causes the build to fail because of the NOTICE checker.
```
Dependency com.fasterxml.jackson.core:jackson-core:2.13.2 is not listed.
Dependency com.fasterxml.jackson.core:jackson-annotations:2.13.2 is not listed.
Dependency com.fasterxml.jackson.core:jackson-databind:2.13.2.2 is not listed.
...
Dependency com.fasterxml.jackson.core:jackson-core:jar:2.13.2 is not bundled, but listed.
Dependency com.fasterxml.jackson.core:jackson-annotations:jar:2.13.2 is not bundled, but listed.
Dependency com.fasterxml.jackson.core:jackson-databind:jar:2.13.2.2 is not bundled, but listed.
```

Reply:
It is not the first time I have done this!